operator/src/main/java/oracle/kubernetes/operator/helpers/JobStepContext.java

```diff
@@ -196,6 +200,7 @@ protected V1JobSpec createJobSpec(TuningParameters tuningParameters) {
 
   private V1PodTemplateSpec createPodTemplateSpec(TuningParameters tuningParameters) {
     V1ObjectMeta metadata = new V1ObjectMeta().name(getJobName());
+    if (istioEnabled()) metadata.putAnnotationsItem("sidecar.istio.io/inject", "false");
```

Comment:
if statements need braces

Reply:
this is forced by the code style checking of mvn plugin.

Reply:
This is a surprise.  I thought the CheckStyle rule expected braces.